admin/api_clusters.go

```diff
@@ -22,8 +22,6 @@ type ClustersApi interface {
 		@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 		@param groupId Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.  **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
 		@return CreateClusterApiRequest
-
-		Deprecated: this method has been deprecated. Please check the latest resource version for ClustersApi
```

Comment:
can't fully understand why is this being deleted? It was deprecated and now not anymore?

Reply:
This PR contains new Atlas API version. Atlas API (and OpenAPI) with the new version shadows previous ones.
Previous version still holds depreciation but since PR relies on the latest version we do not see it here. 

In this particular case someone removed deprecation in OpenAPI.  Will follow up with relevant to check if that is expected. (No details since we are in public repository)